### PR TITLE
File Added :  Assignment 2 & 3

### PR DESCRIPTION
**Pull request by urmilagagare**

### CODE REVIEW

File: urmila/JAVASCRIPT/Assignment2/largest_number.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment2/largest_number.html]: remove extra line

File: urmila/JAVASCRIPT/Assignment2/largest_number.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        <hr style="background-color: white" />
+        <form>
+            <div class="row">
+                <div class="col">
+                    First Number<input type="text" class="form-control" placeholder="First" id="input1"><br>
+                    Second Number<input type="text" class="form-control" placeholder="Second" id="input2"><br>
+                    Third Number<input type="text" class="form-control" placeholder="Third" id="input3"><br>
+                    Forth Number<input type="text" class="form-control" placeholder="Forth" id="input4"><br>
+                    Fifth Number<input type="text" class="form-control" placeholder="Fifth" id="input5"><br>
+                    <button onclick="show()" type="button"
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment2/largest_number.html]: button tag proper indent

File: urmila/JAVASCRIPT/Assignment2/largest_number.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                </div>
+            </div>
+        </form>
+        <p id="p"></p>
+        <p id="pid"></p>
+        <p id="pid1"></p>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+        integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+        crossorigin="anonymous"></script>
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment2/largest_number.html]: script tag proper indent

File: urmila/JAVASCRIPT/Assignment2/largest_number.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-Piv4xVNRyMGpqkS2by6br4gNJ7DXjqk09RmUpJ8jgGtD7zP9yug3goQfGII0yAns"
+        crossorigin="anonymous"></script>
+    </div>
+</body>
+
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment2/largest_number.html]: remove extra line

File: urmila/JAVASCRIPT/Assignment2/largest_number.html [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+</html>
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment2/largest_number.html:77]: add extra line at the end

File: urmila/JAVASCRIPT/Assignment3/Ass3.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+            <div class="input-box">
+                <div class="input-container">
+                    <input type="text" placeholder="rows" id="rownum" />
+                    <input type="text" placeholder="cols" id="colnum" />
+                </div>
+                <button onclick="printPattern()">show pattern</button>
+            </div>
+            <div class="output-box" id="pattern-output"></div>
+        </div>
+        <!-- Prime Number -->
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment3/Ass3.html]: remove comment

File: urmila/JAVASCRIPT/Assignment3/Ass3.html [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+            <p> Prime Number</p>
+            <div class="input-box flex-center">
+                <div class="input-container">
+                    <input type="text" placeholder="number" id="num" />
+                </div>
+                <button onclick="isPrime()">Prime or not</button>
+            </div>
+            <div class="output-box" id="prime-output"></div>
+        </div>
+        <!-- Leap Year -->
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment3/Ass3.html]: remove coment

File: urmila/JAVASCRIPT/Assignment3/Ass3.html [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+            <p>Even or Odd Number</p>
+            <div class="input-box flex-center">
+                <div class="input-container">
+                    <input type="text" placeholder="number" id="even" />
+                </div>
+                <button onclick="isEven()">Even or Odd </button>
+            </div>
+            <div class="output-box" id="even-output"></div>
+        </div>
+        <!-- Factorial -->
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment3/Ass3.html]: remove all comments

File: urmila/JAVASCRIPT/Assignment3/Ass3.js [hunk 9]
```diff
@@ -0,0 +1,78 @@
+/* Code for the Pattern Program */
+const rowNumInput = document.getElementById("rownum");
+const colNumInput = document.getElementById("colnum");
+const patternOutput = document.getElementById("pattern-output");
+
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment3/Ass3.js]: remove extra line

File: urmila/JAVASCRIPT/Assignment3/Ass3.js [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+    let output = "";
+    for (let i = 1; i <= rows; i++) {
+        for (let j = 1; j <= cols; j++) {
+            output += " * ";
+        }
+        output += " <br> ";
+    }
+    patternOutput.innerHTML = output;
+}
+
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment3/Ass3.js]: remove extra line or comment

File: urmila/JAVASCRIPT/Assignment3/Ass3.js [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+/* Code for the Prime number */
+
+function isPrime() {
+    let num = parseInt(document.getElementById("num").value);
+
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment3/Ass3.js]: remove extra line

File: urmila/JAVASCRIPT/Assignment3/Ass3.js [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+            ).innerHTML = `nope ${num} is not prime`;
+            break;
+        } else {
+            document.getElementById(
+                "prime-output"
+            ).innerHTML = ` Yup ${num} is prime`;
+        }
+    }
+}
+
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment3/Ass3.js]: remove extra line

File: urmila/JAVASCRIPT/Assignment3/Ass3.js [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+function isLeap() {
+    let year = parseInt(document.getElementById("year").value);
+
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment3/Ass3.js]: remove extra line

File: urmila/JAVASCRIPT/Assignment3/Ass3.js [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+            "leap-output"
+        ).innerHTML = `nope ${year} is not Leap year`;
+    }
+    if (isNaN(year)) {
+        document.getElementById(
+            "leap-output"
+        ).innerHTML = `nope ${year} is not even a year`;
+    }
+}
+
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment3/Ass3.js]: remove extra line

File: urmila/JAVASCRIPT/Assignment3/Ass3.js [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+function isEven() {
+    let number = parseInt(document.getElementById("even").value);
+
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment3/Ass3.js]: remove extra line

File: urmila/JAVASCRIPT/Assignment3/Ass3.js [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+        document.getElementById(
+            "even-output"
+        ).innerHTML = `${number} is Even Number`;
+    } else {
+        document.getElementById(
+            "even-output"
+        ).innerHTML = `${number} is Odd Number`;
+    }
+}
+
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment3/Ass3.js]: remove all extra lines

File: urmila/JAVASCRIPT/Assignment3/style.css [hunk 17]
```diff
@@ -0,0 +1,82 @@
...
+.output-box {
+    background-color:green;
+    padding: 20px;
+    border-radius: 10px;
+    color: #ffffff;
+    font-family: Poppins;
+    stroke-width: 500;
+   
+}
+/* Leap */
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment3/style.css]: remove comment

File: urmila/JAVASCRIPT/Assignment3/style.css [hunk 18]
```diff
@@ -0,0 +1,82 @@
...
+.prime-container input,
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment3/style.css:75]: this code required in one line

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

**Review by nileshp1122 — APPROVED** [review]

File: urmila/JAVASCRIPT/Assignment2/grade_system.html [hunk 19]
```diff
@@ -0,0 +1,75 @@
...
+                border: 1px solid black;
+                background-image: url("Image.png");
+                margin-top: 150px;
+            }
+            button {
+                margin-top: 30px;
+            }
+        </style>
+    </head>
+      <body>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment2/grade_system.html]: check indentation

File: urmila/JAVASCRIPT/Assignment2/largest_number.html [hunk 20]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment2/largest_number.html:3]: check indentation

File: urmila/JAVASCRIPT/Assignment2/largest_number.html [hunk 21]
```diff
@@ -0,0 +1,73 @@
...
+            else if (five > first && five > second && five > third && five < forth || five > first && five > second && five < third && five > forth || five > fist && five < second && five > third && five > forth || five < first && five > second && five > third && five > forth) {
+                document.getElementById("p").innerHTML = "<h2>Second Largest Number is :</h2>" + "<h2>" + five + "</h2>";
+            }
+            else {
+                document.getElementById("p").innerHTML = "Enter Number!!";
+            }
+        }
+    </script>
+</head>
+    <body>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment2/largest_number.html:48]: check indentation

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**